examples/api/templates/emails/memberSubscriptionPayment/offSessionPaymentFailed/html.pug

```diff
@@ -0,0 +1,7 @@
+extends ../../base.pug
+
+block content
+    h1 Hi #{user.name}
+
+    p We tried to bill your cc with #{paymentProviderID} and received the following error: "#{errorCode}"
```

Comment:
This looks very technical with `#{paymentProviderId}`. Is provider sending email as well?

Reply:
@tomaszdurka this is just as an example anyway. Actual we.publish user could do an `if` depending on the `paymentProviderId`